cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
-
-describe('Student Registration page', () => {
-  before(() => {
-
-  });
-
-  it('', () => {
-
+describe('Automation Practice Form', () => {
+  it('should successfully register a new user with valid data', () => {
+    cy.visit(Cypress.env('form_url'));
+    cy.get('#firstName').type('John');
+    cy.get('#lastName').type('Doe');
+    cy.get('#userEmail').type('john.doe@example.com');
+    cy.get('label[for="gender-radio-1"]').click();
```

Comment:
It would be better to use the actual value (e.g., 'Male') instead of the index (e.g., 'gender-radio-1') for readability and maintainability of the tests. If the order of the options changes, the test might select the wrong option.